docs/topics/getting-started.md

```diff
@@ -166,7 +166,52 @@ Follow Kotlin on ![Twitter](twitter.svg){width=25}{type="joined"} [Twitter](http
 
 </tab>
 
+<tab title="Multiplatform library">
+
+Support for multiplatform programming is one of Kotlin’s key benefits. It reduces time spent writing and maintaining the same code for different platforms while retaining the flexibility and benefits of native programming.
+
+Here you'll learn how to develop and a multiplatform library:
+
+1. **Install the [latest version of IntelliJ IDEA](http://www.jetbrains.com/idea/download/index.html).**
+
+2. **Create a multiplaform library:**
+
+   * To start from scratch, [create a basic project](mpp-create-lib.md).
+   * If you prefer more robust examples, complete the [Create and publish a multiplatform library](multiplatform-library.md) tutorial. It shows how to create a multiplatform library for JVM, JS, and Native platforms, test it and publish to a local Maven repository.
+
+3. **Use libraries in your application.** Learn more about [adding dependencies](mpp-add-dependencies.md).
+
+   |Library|Details|
+   |-------|-------|
+   | Ktor |  [Docs](https://ktor.io/) and [sample](https://kotlinlang.org/docs/mobile/use-ktor-for-networking.html).| 
```

Comment:
In KMM, we call "samples" something that has the code on github and can be downloaded https://kotlinlang.org/docs/mobile/samples.html#peopleinspace. So I wouldn't say that this content by the link you used is a sample indeed. The hands-on tutorial might fit here better, I suppose... https://kotlinlang.org/docs/mobile/complete-networking-data-storage-hands-on.html

But I see that on the line below you also post the link on the same hands-on.

Reply:
I've rearrange some stuff here, now it looks more logical, I guess.